packages/cli/templates/csharp/echo/{{name}}.hbs/MainController.cs.hbs

```diff
@@ -0,0 +1,30 @@
+using Microsoft.Teams.Api.Activities;
+using Microsoft.Teams.Apps;
+using Microsoft.Teams.Apps.Annotations;
+using Microsoft.Teams.Apps.Activities;
+using Microsoft.Teams.Apps.Plugins;
+
+namespace {{name}};
+
+[TeamsController("main")]
+public class MainController
+{
+    [Message]
+    public async Task OnMessage([Context] MessageActivity activity, [Context] IContext.Client client)
+    {
+        await client.Send($"you said \"{activity.Text}\"");
+    }
+
+    [Activity]
```

Comment:
same here, just the top handler is needed

Reply:
ok